.teamcity/settings.kts

```diff
@@ -64,6 +64,8 @@ project {
 		text("docker_image_e2e", "registry.a8c.com/calypso/ci-e2e:latest", label = "Docker e2e image", description = "Docker image used to run e2e tests", allowEmpty = true)
 		text("env.DOCKER_BUILDKIT", "1", label = "Enable Docker BuildKit", description = "Enables BuildKit (faster image generation). Values 0 or 1", allowEmpty = true)
 		password("CONFIG_E2E_ENCRYPTION_KEY", "credentialsJSON:819c139c-90a1-4803-8367-00e5aa5fdb07", display = ParameterDisplay.HIDDEN)
+		password("CONFIG_E2E_ENCRYPTION_KEY_JAN_22", "credentialsJSON:28f1cadd-35d5-44b1-bdca-4228fe2948b7", display = ParameterDisplay.HIDDEN)
+		param("E2E_CONFIG_ENCRYPTION_KEY", "%CONFIG_E2E_ENCRYPTION_KEY_JAN_22%")
```

Comment:
This second value means that consumers can just reference a single config value for e2e keys, and that gets filled with the correct key based on vcs.